SharpBucket/V2/EndPoints/TeamsEndPoint.cs

```diff
@@ -26,40 +23,44 @@ public class TeamsEndPoint{
         }
 
         /// <summary>
-        /// Gets the team's members. By default, this call returns the first 50 members of the team. 
+        /// Gets the team's members.
         /// </summary>
+        /// <param name="max">The maximum number of items to return. 0 returns all items.</param>
         /// <returns></returns>
-        public User ListMembers(){
+        public List<Team> ListMembers(int max = 0){
```

Comment:
I think you can ditch the "Team" object and just use "User", but I'll create a ticket for that.